fdbrpc/TokenSign/TokenSignStdTypes.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * TokenSignStlTypes.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbrpc/TokenSign.h"
+#include "fdbrpc/TokenSignStdTypes.h"
+#include "flow/PKey.h"
+#include "flow/MkCert.h"
+#include <stdexcept>
+
+namespace {
+
+// converts std::optional<STANDARD_TYPE> to Optional<FLOW_TYPE>
+template <class T, class S>
+void convertAndAssign(Arena& arena, Optional<T>& to, const std::optional<S>& from) {
+	if constexpr (std::is_same_v<S, std::vector<std::string>>) {
+		static_assert(std::is_same_v<T, VectorRef<StringRef>>);
```

Comment:
Please add a string description to the static_assert

Reply:
Done!